packages/qwik-city/src/runtime/src/server-functions.ts

```diff
@@ -224,53 +229,73 @@ export const validatorQrl = ((
 /** @public */
 export const validator$: ValidatorConstructor = /*#__PURE__*/ implicit$FirstArg(validatorQrl);
 
+const flattenValibotIssues = (issues: v.GenericIssue[]) => {
+  return issues.reduce<Record<string, string | string[]>>((acc, issue) => {
+    if (issue.path) {
+      const hasArrayType = issue.path.some((path) => path.type === 'array');
+      if (hasArrayType) {
+        const keySuffix = issue.expected === 'Array' ? '[]' : '';
+        const key =
+          issue.path
+            .map((item) => (item.type === 'array' ? '*' : item.key))
```

Comment:
@fabian-hiller I don't know much about valibot, what is that key? Is it a wrong valibot version, that it fails?

Sorry to get back to you guys so late. We had 10 hour shifts fixing the qwik-shop.

Reply:
Which version of Valibot do you use? We will hit v1 RC with a stable API soon.

Reply:
@fabian-hiller none. This might be the problem?

Reply:
I am not an expert on the Qwik repo and cannot help with that. I don't think this "problem" is related to this PR.

Reply:
Okay. I think I know what you mean now. I will take a look.